src/lcm/grids.py

```diff
@@ -93,7 +97,7 @@ def to_jax(self) -> Array:
     def get_coordinate(self, value: Scalar) -> Scalar:
         """Get the generalized coordinate of a value in the grid."""
 
-    def replace(self, **kwargs: Any) -> "ContinuousGrid":  # noqa: ANN401
+    def replace(self, **kwargs: float) -> ContinuousGrid:
```

Comment:
```suggestion
    def replace(self, **kwargs: float | int) -> ContinuousGrid:
```
I could replace `n_points`, right?